eclair-core/src/main/scala/fr/acinq/eclair/Setup.scala

```diff
@@ -188,7 +188,8 @@ class Setup(datadir: File,
           blocks_6 = config.getLong("default-feerates.delay-blocks.6"),
           blocks_12 = config.getLong("default-feerates.delay-blocks.12"),
           blocks_36 = config.getLong("default-feerates.delay-blocks.36"),
-          blocks_72 = config.getLong("default-feerates.delay-blocks.72")
+          blocks_72 = config.getLong("default-feerates.delay-blocks.72"),
+          blocks_144 = config.getLong("default-feerates.delay-blocks.144")
```

Comment:
I'm not sure that a target of 144 blocks is necessary or even realistic, afaict all the estimators that we use will return the same estimates when target > 60 blocks

Reply:
We could have 144 to be the fallback-lowest fee applicable even if the estimator doesn't give an exact estimate for more than 60 blocks.